packages/x-date-pickers/src/themeAugmentation/themeAugmentation.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { createTheme } from '@mui/material/styles';
+
+createTheme({
+  components: {
+    MuiDatePicker: {
```

Comment:
Should we cover other components here as well?

Reply:
Not sure it's supper usefull since they are all similar, but it does not hurt to do it